discovery_engine_core/web-api/openapi.yaml

```diff
@@ -0,0 +1,160 @@
+openapi: 3.0.3
+info:
+  title: Personalization API
+  version: 1.0.0-rc1
+  description: |-
+    The endpoints can be contacted at `t<name>.xaynet.dev/v1` or `t<name>.xaynet.dev/v1.0`.
```

Comment:
`t` for `tenant` but could also be different. I think it could be useful for us to have some kind of template in case we want to automatize some part of the domain.

Reply:
Removed.